### ISSUE TITLE
iOS App - No download or upload of media (Pictures, etc.)

### ISSUE THREAD

**Issue by a7am** [actorapp/actor-platform#930]
The iOS app in latest platform version does not show, download or upload any media such as pictures.
The console log shows the following when user tries to access media:
`Jun  8 18:46:11 kherad ActorApp[1857] <Warning>: ApiBroker: -> request#10: rpc GetFileUploadPartUrl{uploadKey=43dda1245ce6b2502a3af0e749a2c1a2}
Jun  8 18:46:11 kherad ActorApp[1857] <Warning>: ApiBroker: <- response#10: tuple GetFileUploadPartUrl{} in 84 ms`

And also I saw this:

`Jun  8 19:29:01 kherad ActorApp[209] <Warning>: WARRING ApiBroker: <- error#54: PHONE_CODE_EXPIRED 400 Code is expired. in 423 ms`

Any ideas?

**Comment by rockjam** [comment]
@a7am Are we speaking about your own self-hosted server?

**Comment by a7am** [comment]
Yes, this app is connected to my self-hosted server, but the bootstrap app which is connected to the same server does not have this problem.
I have also talked about another issue with this platform in #926, can these two be connected? I mean project signing and the error here 'phone code expired'

**Comment by rockjam** [comment]
@a7am do you have files and media working in android and web app?

**Comment by a7am** [comment]
Yes, sending files and media are working in platform version of android app, but I don't have the web app to test it.
